gcovr/configuration.py

```diff
@@ -64,6 +65,15 @@ def check_input_file(value, basedir=None):
     return os.path.abspath(value)
 
 
+def timestamp(value: str) -> datetime.datetime:
+    from .timestamps import parse_timestamp  # lazy import
+    try:
+        return parse_timestamp(value)
+    except ValueError as ex:
```

Comment:
```suggestion
    except ValueError as ex:  # pragma: no cover
```

Reply:
I added a test for the exception handler (and promptly found a bug with the exception formatting :grimacing:)